backend/graphql/schema.graphql

```diff
@@ -64,13 +64,12 @@ type User {
 Create a new user.
 """
```

Comment:
Shouldn't we remove this comment if it's being added within the `Mutation` object?

```suggestion

```

Side Note: I tried doing [a multiline suggestion](https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/) but it's not working for me.

Reply:
You are correct. 👍